app/routes/products.$categorySlug/route.module.scss

```diff
@@ -98,8 +105,19 @@
     }
 }
 
+@media (max-width: $tablet-width) {
+    .countAndSorting {
+        flex-direction: column;
+        align-items: end;
+    }
+
+    .productsCount {
+        align-self: start;
+    }
+}
+
 @media (max-width: $mobile-width) {
-    .navigation {
+    .sidebar {
```

Comment:
Not related to PR, but it's a small thing I forgot to change when added filters.